src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -89,3 +91,4 @@ def load_arguments(self, _):
         c.argument('context_name', options_list=['--kube-context'], help='If specified, overwrite the default context name.')
         c.argument('path', options_list=['--file', '-f'], type=file_type, completer=FilesCompleter(), default=os.path.join(os.path.expanduser('~'), '.kube', 'config'), help="Kubernetes configuration file to update. If not provided, updates the file '~/.kube/config'. Use '-' to print YAML to stdout instead.")
         c.argument('api_server_port', options_list=['--port'], help='Port used for accessing connected cluster.')
+        c.argument('namespace', options_list=['--namespace'], help='Cluster Credentials are retrieved at this namespace scope.')
```

Comment:
Should we modify the log of "connectedk8s proxy" command to communicate to user that we're fetching clusterCreds at namespace scope when this param is given?

Reply:
The kubeconfig which is returned is the same one which would be returned if called at the cluster scope. The only difference is the api is called at the scope of the namespace resource. I don't think the logs would need to be changed, since the end result is the same either way, this is just internal implementation differences, right?